docfx.json

```
@@ -155,8 +155,7 @@
         "docs/framework/xaml-services/**/**.md": "None"
       },
       "feedback_product_url": {
-        "docs/azure/sdk/**/**.md": "https://github.com/azure/azure-sdk-for-net",
-        "docs/azure/sdk/**/**.yml": "https://github.com/azure/azure-sdk-for-net",
+        "docs/azure/sdk/**/*.*": "https://github.com/azure/azure-sdk-for-net",
```

I'm seeing ms.topic have entries like:

```
        "docs/csharp/programming-guide/**" : "conceptual",
```

Is it valid if I used the following here, and the other entries changed in this PR?

```suggestion
        "docs/azure/sdk/**/": "https://github.com/azure/azure-sdk-for-net",
```